app/charges/controllers.py

```diff
@@ -246,19 +234,13 @@ def edit_charge(user, user_data):
     committee = Committees.query.filter_by(id = charge.committee).first()
     membership = committee.members.filter_by(member= user).first()
 
-    if (user.id != committee.head and user.is_admin == False and
-        (membership is None or membership.role != Roles.ActiveMember)):
-        emit("edit_charge", Response.PermError)
-        return
-
-    # Only admins and committee heads can make charges public.
```

Comment:
The logic was changed so that only CommitteeHeads or admins could edit charges at all. Therefore we don't need to check what specific changes are being made here.